package.json

```diff
@@ -38,6 +38,7 @@
     "express": "^4.11.2",
     "jison": "0.4.17",
     "logops": "1.0.0",
+    "moment": "^2.19.4",
```

Comment:
Which is using this new module? A link to the point of code in which it is use would be clarifying...

Reply:
It is used here: https://github.com/dcalvoalonso/iotagent-node-lib/blob/task/bidirectionalityPlugin/lib/services/ngsi/subscriptionService.js#L228.

In NGSIv1, the throttling was specified using the ISO 8601 standard format (e.g., PT5S). However, in NGSIv2, the throttling is just an Integer in seconds. Therefore, we use this module to do the conversion.

Reply:
NTC